lib/core/src/lib/layout/components/layout-container/layout-container.component.scss

```diff
@@ -37,13 +38,14 @@ adf-layout-container {
 }
 
 /* stylelint-disable selector-class-pattern */
-mat-sidenav-content.mat-sidenav-content,
-.mat-drawer-transition .mat-drawer-content {
+mat-sidenav-content.mat-sidenav-content.mat-drawer-content,
```

Comment:
the same question here

Reply:
not sure if that's possible, I think there is a custom hiding mechanism ex. toggledContentAnimation() in layout container component

Reply:
just like we talked on Slack, I will leave it just in case if it is being used somewhere